build.js

```diff
@@ -269,7 +268,7 @@ process.nextTick(function () {
         fs.writeFileSync(fpath, code);
         var output;
         try {
-          output = "" + child_process.execSync('node_modules/closurecompiler/bin/ccjs ' +
+          output = "" + child_process.execSync('./node_modules/.bin/google-closure-compiler-js ' +
```

Comment:
we shouldn't ever be (or have been) hardcoding `node_modules` - let's use `which('google-closure-compiler-js')` and https://www.npmjs.com/package/npm-which instead?